tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -44,7 +44,8 @@ def run(self):
         # To detect out of band deletes
         # refresh gluster object inventory at config['sync_interval']
         # Default is 260 seconds
-        SYNC_TTL = int(NS.config.data.get("sync_interval", 10)) + 250
+        SYNC_TTL = int(NS.config.data.get("sync_interval", 10)) +
+                   int(NS.config.data.get("ttl_delta", 600))
```

Comment:
I would prefer "ttl_delta" to not be exposed to user. 

ttl_delta should be calculated on the count of volume and bricks

Reply:
Done. Will send another PR for the same.